### PR TITLE
Update performance test cert and add to DPR

### PR DESCRIPTION
**Pull request by cthayes**
This PR updates the performance test cert, since we no longer have access to the private key. It also adds the cert to the DPR keystore.

BLUEBUTTON-758
BLUEBUTTON-220

### CODE REVIEW

File: files/client_data_server_performance_tester_certificate.pem [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+J0G2thQccUXS1If3kcD6S9bv0uqI8INFoOZKzRXYxCXEuOF9E+E4NOqzwqBqnbVz
+9WyT6wHUs/NPVbSD7MQcXud25CZ7pMhV29pJWjSIJdK5r+/H/9PfeGthaBjoiSR5
+g0edNdsVybDtrLPq8wUYehWYnt/RbS1l6OYPzeuzh93sUerZ3dTHfFsilCaVvnod
+nkregDKVW6jk/sMd8KeIGrdDSy5+AAoeU5ORKG4Sk168v0j609HbKrJyu77wYqo3
+GxYfI2MRPcerCgqpROXpSgDrmAP9ongBlyTrhimvTa5glzZjmSFbhbrQXgWzGgnR
+XugoDGtjNDCy4SfI7QLQKWvqXcxZbDACFXPeGFA8TvFBFTYxeC5r3Dh2CsFgt/pP
+VgKcUVQICZI7onzZHq+zpKpaPd/s/evyR8Mau8jhSHtLPlR3RcTdZOIDnWs3HEdJ
+26gDAiX0BerZghS4KuHPTRRhYAJWiV3BN6CVWNvkQAJFYOeY6zLPspxtu2JZbWVo
+waUfnyPo4LKjCl6vrf1afbvIbJuzLribuNi7zfWp6g==
+-----END CERTIFICATE-----
```
Comment (karlmdavis) [inline files/client_data_server_performance_tester_certificate.pem]:
```
$ openssl x509 -in files/client_data_server_performance_tester_certificate.pem -text
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 1153466444 (0x44c0804c)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=client-test
        Validity
            Not Before: Sep  8 23:28:25 2017 GMT
            Not After : Sep  6 23:28:25 2027 GMT
        Subject: CN=client-test
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:98:56:c5:69:37:72:ad:06:fb:62:c4:01:01:a1:
                    e1:5c:29:60:5f:da:bb:1b:fa:9d:93:f5:b6:73:26:
                    ef:36:98:b3:10:e7:a8:05:94:d4:36:81:51:1f:4c:
                    be:28:ad:cb:31:c2:c4:5d:47:3d:fc:f6:55:0e:60:
                    ca:d0:8a:a7:5e:a5:4a:db:4f:75:ba:47:5c:8a:20:
                    e2:75:bb:93:a2:ea:83:08:34:30:58:b6:f9:cd:d6:
                    e7:20:f8:23:51:7e:c4:f8:02:9a:a2:7d:4a:e1:5b:
                    98:e2:8f:fb:7b:33:c6:9b:a0:55:ad:7d:90:7a:be:
                    a9:84:40:86:36:ae:a0:b2:a1:2a:10:a4:6a:d0:73:
                    ab:98:2e:69:3d:3a:37:b2:67:2d:60:f8:44:d0:35:
                    f5:5b:04:e2:1b:3f:58:9b:fb:8c:92:67:5c:c7:59:
                    cf:b3:01:e5:5a:99:3d:c0:39:d1:ee:25:65:90:b8:
                    66:e0:aa:94:97:e6:b5:fb:85:4f:1c:60:71:bb:56:
                    1a:ef:8e:8c:90:75:37:41:31:16:a5:85:b0:5c:e5:
                    49:b1:ab:91:76:ad:38:18:fc:4a:92:ff:a1:68:9e:
                    a0:94:92:37:2b:4f:03:3d:93:96:8b:e1:fa:42:c7:
                    9b:78:26:55:11:36:d2:0c:ba:21:96:cb:72:b7:c8:
                    60:da:69:69:a1:89:86:81:09:eb:27:28:b0:bb:ad:
                    52:3a:15:69:01:04:fd:7d:30:dc:0f:a2:d6:93:32:
                    17:c8:01:a8:39:fa:1f:1e:cd:7d:9d:92:6b:89:f7:
                    e8:2c:ad:38:7c:37:f0:73:f9:df:17:c6:84:66:5f:
                    04:92:33:95:28:8c:1f:d5:88:08:11:35:e6:9b:48:
                    d0:80:a9:fd:d2:79:d7:5f:f0:d6:04:44:8c:91:e1:
                    de:42:4f:95:72:dc:bf:38:58:76:96:46:39:59:da:
                    77:99:d7:3e:bc:cc:26:19:ae:c7:e4:9c:48:20:51:
                    bd:04:2a:67:96:32:85:47:61:08:31:d5:f7:96:77:
                    6c:de:3c:f7:e5:4b:f4:33:bf:dc:18:bc:d5:a3:54:
                    7e:05:ff:13:98:53:a4:2c:95:69:3a:7c:71:a1:66:
                    91:35:7d:a8:e5:14:c9:f3:7c:27:97:8b:24:d1:ee:
                    46:dd:21:ea:ad:d6:85:54:75:69:ed:d4:75:45:0d:
                    07:e9:cb:45:16:a8:5e:03:85:9a:38:2a:b3:4e:bc:
                    cb:e4:33:78:9b:21:24:b9:7d:91:3d:b0:6a:9e:f7:
                    a7:87:65:99:8a:3a:96:46:f1:e0:9f:12:93:32:44:
                    e2:a4:21:1f:be:07:4f:e7:cd:12:9a:f9:5e:e3:9b:
                    2c:f4:d7
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Subject Key Identifier:
                81:1E:48:4A:FE:BA:12:A0:36:E1:3B:68:56:55:8C:7C:B0:B4:26:C7
    Signature Algorithm: sha256WithRSAEncryption
         92:92:64:ad:29:ed:d4:38:e5:c9:3c:b3:32:14:de:07:6a:ac:
         8a:13:51:18:f1:7c:69:43:46:89:b3:4e:01:a6:e2:e5:cb:1e:
         29:dc:31:3e:de:c2:8a:01:a4:57:2b:15:fa:dc:16:0c:5c:77:
         50:98:5c:e7:70:9e:fd:4c:cf:bc:0c:9b:6f:d3:0f:0d:bb:d6:
         ae:d8:95:e0:06:61:c5:80:2b:76:53:9e:89:5d:44:3f:a4:10:
         b7:1c:60:b5:08:94:f4:27:41:b6:b6:14:1c:71:45:d2:d4:87:
         f7:91:c0:fa:4b:d6:ef:d2:ea:88:f0:83:45:a0:e6:4a:cd:15:
         d8:c4:25:c4:b8:e1:7d:13:e1:38:34:ea:b3:c2:a0:6a:9d:b5:
         73:f5:6c:93:eb:01:d4:b3:f3:4f:55:b4:83:ec:c4:1c:5e:e7:
         76:e4:26:7b:a4:c8:55:db:da:49:5a:34:88:25:d2:b9:af:ef:
         c7:ff:d3:df:78:6b:61:68:18:e8:89:24:79:83:47:9d:35:db:
         15:c9:b0:ed:ac:b3:ea:f3:05:18:7a:15:98:9e:df:d1:6d:2d:
         65:e8:e6:0f:cd:eb:b3:87:dd:ec:51:ea:d9:dd:d4:c7:7c:5b:
         22:94:26:95:be:7a:1d:9e:4a:de:80:32:95:5b:a8:e4:fe:c3:
         1d:f0:a7:88:1a:b7:43:4b:2e:7e:00:0a:1e:53:93:91:28:6e:
         12:93:5e:bc:bf:48:fa:d3:d1:db:2a:b2:72:bb:be:f0:62:aa:
         37:1b:16:1f:23:63:11:3d:c7:ab:0a:0a:a9:44:e5:e9:4a:00:
         eb:98:03:fd:a2:78:01:97:24:eb:86:29:af:4d:ae:60:97:36:
         63:99:21:5b:85:ba:d0:5e:05:b3:1a:09:d1:5e:e8:28:0c:6b:
         63:34:30:b2:e1:27:c8:ed:02:d0:29:6b:ea:5d:cc:59:6c:30:
         02:15:73:de:18:50:3c:4e:f1:41:15:36:31:78:2e:6b:dc:38:
         76:0a:c1:60:b7:fa:4f:56:02:9c:51:54:08:09:92:3b:a2:7c:
         d9:1e:af:b3:a4:aa:5a:3d:df:ec:fd:eb:f2:47:c3:1a:bb:c8:
         e1:48:7b:4b:3e:54:77:45:c4:dd:64:e2:03:9d:6b:37:1c:47:
         49:db:a8:03:02:25:f4:05:ea:d9:82:14:b8:2a:e1:cf:4d:14:
         61:60:02:56:89:5d:c1:37:a0:95:58:db:e4:40:02:45:60:e7:
         98:eb:32:cf:b2:9c:6d:bb:62:59:6d:65:68:c1:a5:1f:9f:23:
         e8:e0:b2:a3:0a:5e:af:ad:fd:5a:7d:bb:c8:6c:9b:b3:2e:b8:
         9b:b8:d8:bb:cd:f5:a9:ea
-----BEGIN CERTIFICATE-----
MIIEyzCCArOgAwIBAgIERMCATDANBgkqhkiG9w0BAQsFADAWMRQwEgYDVQQDEwtj
bGllbnQtdGVzdDAeFw0xNzA5MDgyMzI4MjVaFw0yNzA5MDYyMzI4MjVaMBYxFDAS
BgNVBAMTC2NsaWVudC10ZXN0MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKC
AgEAmFbFaTdyrQb7YsQBAaHhXClgX9q7G/qdk/W2cybvNpizEOeoBZTUNoFRH0y+
KK3LMcLEXUc9/PZVDmDK0IqnXqVK2091ukdciiDidbuTouqDCDQwWLb5zdbnIPgj
UX7E+AKaon1K4VuY4o/7ezPGm6BVrX2Qer6phECGNq6gsqEqEKRq0HOrmC5pPTo3
smctYPhE0DX1WwTiGz9Ym/uMkmdcx1nPswHlWpk9wDnR7iVlkLhm4KqUl+a1+4VP
HGBxu1Ya746MkHU3QTEWpYWwXOVJsauRdq04GPxKkv+haJ6glJI3K08DPZOWi+H6
QsebeCZVETbSDLohlstyt8hg2mlpoYmGgQnrJyiwu61SOhVpAQT9fTDcD6LWkzIX
yAGoOfofHs19nZJriffoLK04fDfwc/nfF8aEZl8EkjOVKIwf1YgIETXmm0jQgKn9
0nnXX/DWBESMkeHeQk+Vcty/OFh2lkY5Wdp3mdc+vMwmGa7H5JxIIFG9BCpnljKF
R2EIMdX3lnds3jz35Uv0M7/cGLzVo1R+Bf8TmFOkLJVpOnxxoWaRNX2o5RTJ83wn
l4sk0e5G3SHqrdaFVHVp7dR1RQ0H6ctFFqheA4WaOCqzTrzL5DN4myEkuX2RPbBq
nvenh2WZijqWRvHgnxKTMkTipCEfvgdP580Smvle45ss9NcCAwEAAaMhMB8wHQYD
VR0OBBYEFIEeSEr+uhKgNuE7aFZVjHywtCbHMA0GCSqGSIb3DQEBCwUAA4ICAQCS
kmStKe3UOOXJPLMyFN4HaqyKE1EY8XxpQ0aJs04BpuLlyx4p3DE+3sKKAaRXKxX6
3BYMXHdQmFzncJ79TM+8DJtv0w8Nu9au2JXgBmHFgCt2U56JXUQ/pBC3HGC1CJT0
J0G2thQccUXS1If3kcD6S9bv0uqI8INFoOZKzRXYxCXEuOF9E+E4NOqzwqBqnbVz
9WyT6wHUs/NPVbSD7MQcXud25CZ7pMhV29pJWjSIJdK5r+/H/9PfeGthaBjoiSR5
g0edNdsVybDtrLPq8wUYehWYnt/RbS1l6OYPzeuzh93sUerZ3dTHfFsilCaVvnod
nkregDKVW6jk/sMd8KeIGrdDSy5+AAoeU5ORKG4Sk168v0j609HbKrJyu77wYqo3
GxYfI2MRPcerCgqpROXpSgDrmAP9ongBlyTrhimvTa5glzZjmSFbhbrQXgWzGgnR
XugoDGtjNDCy4SfI7QLQKWvqXcxZbDACFXPeGFA8TvFBFTYxeC5r3Dh2CsFgt/pP
VgKcUVQICZI7onzZHq+zpKpaPd/s/evyR8Mau8jhSHtLPlR3RcTdZOIDnWs3HEdJ
26gDAiX0BerZghS4KuHPTRRhYAJWiV3BN6CVWNvkQAJFYOeY6zLPspxtu2JZbWVo
waUfnyPo4LKjCl6vrf1afbvIbJuzLribuNi7zfWp6g==
-----END CERTIFICATE-----
```

Things I'm checking:

1. There's no private key material here, right? Correct.
2. Is the cert different from any other certs from the same project (just watching out for oopses)? Yes.
3. Does it have a reasonably useful/informative CN? Nope: "`CN=client-test`" isn't very useful.
4. Is its `Subject` unique? Kinda'.
5. Does it have a reasonable key size (>= 2048)? Yes.

Could you re-issue it with a new `Subject` (and prolly `Issuer`)? I'd suggest something like "`CN=bfs-performance-tests`", but YMMV.

Comment (cthayes) [reply files/client_data_server_performance_tester_certificate.pem]: I just updated the cert to the one that was already in the perf tests repo, but I'll create a new one.

File: files/client_data_server_performance_tester_certificate.pem [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+ngYDVR0jBIGWMIGTgBQhre0xNeHp2Ncr1AaoUPemp+NDmaFwpG4wbDELMAkGA1UE
+BhMCVVMxETAPBgNVBAgTCE1hcnlsYW5kMRIwEAYDVQQHEwlCYWx0aW1vcmUxDDAK
+BgNVBAoTA0NNUzEMMAoGA1UECxMDQkZTMRowGAYDVQQDExFQZXJmb3JtYW5jZSBU
+ZXN0c4IJALOr3EprlQkNMAkGA1UdEwQCMAAwDQYJKoZIhvcNAQELBQADggEBAKAy
+s4US3sqxy6bjcJnZTetvAxHGvbX2qvzFVRrHKMOpJbiRQXzk+tJkmwnBxkGpYOSF
+oYkUOQom4X+55OZLsSf+OZyw/nKT9P87ETPeGc7EnCWRwA5uW7rWzsuVbiD5fj/E
+7nFONzS/9HAeiA5pB0PCEhUxn5nIQf8zZjkwFtIOR+QeVMzfIoLaxHGh5Q+pfxmV
+AAp2xQqVBH7/rKSp8vNp0FN0vcrAm3cvYqRgLX3bRkMjbOkHN1ETtFRhAJQ095mM
+2A4JGl9KxtrXpP9mI6jTLsNsLtgkllbIRLySTl7kcqpQW+chTsHTupIxeLsi4V0m
+qN983wo6kAit3mgkEAU=
```
Comment (karlmdavis) [inline files/client_data_server_performance_tester_certificate.pem]:
```
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 12946683765997242637 (0xb3abdc4a6b95090d)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, ST=Maryland, L=Baltimore, O=CMS, OU=BFS, CN=Performance Tests
        Validity
            Not Before: Jul 23 19:35:25 2019 GMT
            Not After : Jul 20 19:35:25 2029 GMT
        Subject: C=US, ST=Maryland, L=Baltimore, O=CMS, OU=BFS, CN=Performance Tests
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (2048 bit)
                Modulus:
                    00:d5:59:0e:37:b2:ce:df:06:7e:40:ce:80:c6:60:
                    0e:a3:42:f0:78:c5:db:22:a7:4f:92:ce:0b:c0:59:
                    f5:4b:64:52:76:3d:0c:0b:8b:57:74:da:a2:6c:36:
                    e6:25:5d:4d:b2:2d:1d:2a:42:f9:cb:d2:30:a2:8e:
                    13:16:e9:fd:f2:d3:7b:30:c9:f8:9b:31:93:37:16:
                    93:08:3e:ba:5b:2c:08:ad:73:31:02:77:db:2e:73:
                    b5:18:7f:ee:bf:a4:5c:ac:a8:e0:45:6d:54:41:35:
                    d6:fc:69:7c:2c:27:0d:86:cd:7e:23:e7:da:06:5f:
                    a2:89:06:5e:5e:15:8a:52:45:83:29:8a:cb:11:b3:
                    7e:01:f8:e2:e2:4d:92:b6:7a:77:53:8f:16:f0:50:
                    ed:0b:5b:44:60:fe:f5:97:6f:2b:a5:ab:e6:f0:4e:
                    58:d5:da:99:d4:6c:7f:70:52:15:09:73:ac:e1:5d:
                    e5:dd:54:6c:3d:28:a6:fc:84:72:f8:96:f8:16:ff:
                    6a:1e:14:f6:67:d2:cc:4d:9e:a6:f9:b0:f7:8d:9f:
                    26:1a:13:06:8a:a1:89:4e:d2:a0:62:ec:d3:87:7c:
                    9e:13:8b:dc:98:43:b6:86:46:ef:d6:db:52:34:25:
                    94:6e:e6:35:f1:f7:4b:84:55:71:ee:2a:40:fe:af:
                    c6:af
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Subject Key Identifier:
                21:AD:ED:31:35:E1:E9:D8:D7:2B:D4:06:A8:50:F7:A6:A7:E3:43:99
            X509v3 Authority Key Identifier:
                keyid:21:AD:ED:31:35:E1:E9:D8:D7:2B:D4:06:A8:50:F7:A6:A7:E3:43:99
                DirName:/C=US/ST=Maryland/L=Baltimore/O=CMS/OU=BFS/CN=Performance Tests
                serial:B3:AB:DC:4A:6B:95:09:0D

            X509v3 Basic Constraints:
                CA:FALSE
    Signature Algorithm: sha256WithRSAEncryption
         a0:32:b3:85:12:de:ca:b1:cb:a6:e3:70:99:d9:4d:eb:6f:03:
         11:c6:bd:b5:f6:aa:fc:c5:55:1a:c7:28:c3:a9:25:b8:91:41:
         7c:e4:fa:d2:64:9b:09:c1:c6:41:a9:60:e4:85:a1:89:14:39:
         0a:26:e1:7f:b9:e4:e6:4b:b1:27:fe:39:9c:b0:fe:72:93:f4:
         ff:3b:11:33:de:19:ce:c4:9c:25:91:c0:0e:6e:5b:ba:d6:ce:
         cb:95:6e:20:f9:7e:3f:c4:ee:71:4e:37:34:bf:f4:70:1e:88:
         0e:69:07:43:c2:12:15:31:9f:99:c8:41:ff:33:66:39:30:16:
         d2:0e:47:e4:1e:54:cc:df:22:82:da:c4:71:a1:e5:0f:a9:7f:
         19:95:00:0a:76:c5:0a:95:04:7e:ff:ac:a4:a9:f2:f3:69:d0:
         53:74:bd:ca:c0:9b:77:2f:62:a4:60:2d:7d:db:46:43:23:6c:
         e9:07:37:51:13:b4:54:61:00:94:34:f7:99:8c:d8:0e:09:1a:
         5f:4a:c6:da:d7:a4:ff:66:23:a8:d3:2e:c3:6c:2e:d8:24:96:
         56:c8:44:bc:92:4e:5e:e4:72:aa:50:5b:e7:21:4e:c1:d3:ba:
         92:31:78:bb:22:e1:5d:26:a8:df:7c:df:0a:3a:90:08:ad:de:
         68:24:10:05
-----BEGIN CERTIFICATE-----
MIIEKjCCAxKgAwIBAgIJALOr3EprlQkNMA0GCSqGSIb3DQEBCwUAMGwxCzAJBgNV
BAYTAlVTMREwDwYDVQQIEwhNYXJ5bGFuZDESMBAGA1UEBxMJQmFsdGltb3JlMQww
CgYDVQQKEwNDTVMxDDAKBgNVBAsTA0JGUzEaMBgGA1UEAxMRUGVyZm9ybWFuY2Ug
VGVzdHMwHhcNMTkwNzIzMTkzNTI1WhcNMjkwNzIwMTkzNTI1WjBsMQswCQYDVQQG
EwJVUzERMA8GA1UECBMITWFyeWxhbmQxEjAQBgNVBAcTCUJhbHRpbW9yZTEMMAoG
A1UEChMDQ01TMQwwCgYDVQQLEwNCRlMxGjAYBgNVBAMTEVBlcmZvcm1hbmNlIFRl
c3RzMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA1VkON7LO3wZ+QM6A
xmAOo0LweMXbIqdPks4LwFn1S2RSdj0MC4tXdNqibDbmJV1Nsi0dKkL5y9Iwoo4T
Fun98tN7MMn4mzGTNxaTCD66WywIrXMxAnfbLnO1GH/uv6RcrKjgRW1UQTXW/Gl8
LCcNhs1+I+faBl+iiQZeXhWKUkWDKYrLEbN+Afji4k2Stnp3U48W8FDtC1tEYP71
l28rpavm8E5Y1dqZ1Gx/cFIVCXOs4V3l3VRsPSim/IRy+Jb4Fv9qHhT2Z9LMTZ6m
+bD3jZ8mGhMGiqGJTtKgYuzTh3yeE4vcmEO2hkbv1ttSNCWUbuY18fdLhFVx7ipA
/q/GrwIDAQABo4HOMIHLMB0GA1UdDgQWBBQhre0xNeHp2Ncr1AaoUPemp+NDmTCB
ngYDVR0jBIGWMIGTgBQhre0xNeHp2Ncr1AaoUPemp+NDmaFwpG4wbDELMAkGA1UE
BhMCVVMxETAPBgNVBAgTCE1hcnlsYW5kMRIwEAYDVQQHEwlCYWx0aW1vcmUxDDAK
BgNVBAoTA0NNUzEMMAoGA1UECxMDQkZTMRowGAYDVQQDExFQZXJmb3JtYW5jZSBU
ZXN0c4IJALOr3EprlQkNMAkGA1UdEwQCMAAwDQYJKoZIhvcNAQELBQADggEBAKAy
s4US3sqxy6bjcJnZTetvAxHGvbX2qvzFVRrHKMOpJbiRQXzk+tJkmwnBxkGpYOSF
oYkUOQom4X+55OZLsSf+OZyw/nKT9P87ETPeGc7EnCWRwA5uW7rWzsuVbiD5fj/E
7nFONzS/9HAeiA5pB0PCEhUxn5nIQf8zZjkwFtIOR+QeVMzfIoLaxHGh5Q+pfxmV
AAp2xQqVBH7/rKSp8vNp0FN0vcrAm3cvYqRgLX3bRkMjbOkHN1ETtFRhAJQ095mM
2A4JGl9KxtrXpP9mI6jTLsNsLtgkllbIRLySTl7kcqpQW+chTsHTupIxeLsi4V0m
qN983wo6kAit3mgkEAU=
-----END CERTIFICATE-----
```

Comment (karlmdavis) [reply files/client_data_server_performance_tester_certificate.pem]: Things I'm checking:

1. There's no private key material here, right? Correct.
2. Is the cert different from any other certs from the same project (just watching out for oopses)? Yes.
3. Does it have a reasonably useful/informative CN? Yes.
4. Is its `Subject` unique? Yes.
5. Does it have a reasonable key size (>= 2048)? Yes.

**Pull request merged**